app/viewmodels/checkAnswers/ImportedPlasticPackagingSummary.scala

```diff
@@ -24,14 +24,14 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-object ImportedPlasticPackagingSummary extends SummaryViewModel {
+class ImportedPlasticPackagingSummary(key: String ) extends SummaryViewModel {
```

Comment:
as discussed, lets turn these to private classes, with static objects in the companion

Reply:
done